src/sentry/grouping/enhancer/matchers.py

```diff
@@ -64,7 +64,7 @@ def create_match_frame(frame_data: dict, platform: Optional[str]) -> dict:
         category=get_path(frame_data, "data", "category"),
         family=get_behavior_family_for_platform(frame_data.get("platform") or platform),
         function=_get_function_name(frame_data, platform),
-        in_app=frame_data.get("in_app"),
+        in_app=frame_data.get("in_app") or False,
```

Comment:
`in_app` is explicitly None if grouping runs a second time:
https://github.com/getsentry/sentry/blob/fcb3d2ba785f55ddfb62864b564f5a379f5a522a/src/sentry/stacktraces/processing.py#L267